gradio/components/chatbot.py

```diff
@@ -76,6 +77,7 @@ def __init__(
             show_share_button: If True, will show a share icon in the corner of the component that allows user to share outputs to Hugging Face Spaces Discussions. If False, icon does not appear. If set to None (default behavior), then the icon appears if this Gradio app is launched on Spaces, but not otherwise.
             show_copy_button: If True, will show a copy button for each chatbot message.
             avatar_images: Tuple of two avatar image paths or URLs for user and bot (in that order). Pass None for either the user or bot image to skip. Must be within the working directory of the Gradio app or an external URL.
+            disable_html_sanitization: If True, will disable HTML sanitization for chatbot messages. This is not recommended, as it can lead to security vulnerabilities.
```

Comment:
```suggestion
            disable_html_sanitization: If True, will disable HTML sanitization for chatbot messages. This is not generally recommended, as it can lead to cross-site scripting security vulnerabilities.
```